aws-ts-vpc-with-ecs-fargate-py/ecs-fargate-python/__main__.py

```diff
@@ -0,0 +1,174 @@
+"""An AWS fargate with crosswalk vpc"""
+
+import pulumi
+from pulumi import export, ResourceOptions, Config, StackReference, get_stack, get_project
+import json
+import pulumi_aws as aws
+
+#https://www.pulumi.com/docs/intro/concepts/organizing-stacks-projects/#inter-stack-dependencies
+
+# read local config settings - network
+config = Config()
+
+# reading in vpc StackReference Path from local config
+mystackpath = config.get("mystackpath")
+
+# setting the StackReference
+#mycrosswalkvpc = StackReference(f"{mystackpath}")
```

Comment:
Let's remove the commented code entirely as it's not needed.

```suggestion
```

Reply:
fixed by removing it.

Reply:
Fixed the .get to .require